snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/emitter/EmitterEvent.java

```diff
@@ -0,0 +1,24 @@
+package com.snowplowanalytics.snowplow.tracker.emitter;
+
+import android.support.annotation.NonNull;
+
+import com.snowplowanalytics.snowplow.tracker.payload.Payload;
+
+public class EmitterEvent {
+
+    public final Payload payload;
+    public final long eventId;
+
+    public EmitterEvent(@NonNull Payload payload, long eventId) {
+        this.payload = payload;
+        this.eventId = eventId;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
toString on this object runs the risk of exposing data into logging which might not be desirable. By implementing toString in this way, it becomes much easier for this to happen by mistake - when somewhere in the application the event is used as a string and unexpectadly writes this data in a log to some other system (potentially a third party).
It would be better in my opinion to remove this and directly use the properties (I think this toString is only really used in the tests from what I can see). 
It's a subtle change but one that reduces the footprint for data to leak out on errors/logging.